### PR TITLE
Updated database schema

### PR DESCRIPTION
**Pull request by AvlWx2014**
Adds a column to the `submission_viewing_events` table called `total_time`, which we can use to accumulate total time per link, per round.

### CODE REVIEW

**Review by boergerj — APPROVED** [review]

File: db/migrate/20210315184532_update_review_time_schema.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UpdateReviewTimeSchema < ActiveRecord::Migration
+  def change
+    add_column :submission_viewing_events, :total_time, :integer, :default => 0, column_options: { null: false }
```
Comment (boergerj) [inline db/migrate/20210315184532_update_review_time_schema.rb:3]: Did we want this to be an integter? Or something else?

**Review by boergerj — APPROVED** [review]

File: db/migrate/20210315184532_update_review_time_schema.rb (hunk 1 above)
Comment (AvlWx2014) [reply db/migrate/20210315184532_update_review_time_schema.rb:3]: What else would it be?

Comment (boergerj) [reply db/migrate/20210315184532_update_review_time_schema.rb:3]: I was thinking long based on the diagram in Slack. Either works though. I'm fine with it being integer

Comment (AvlWx2014) [reply db/migrate/20210315184532_update_review_time_schema.rb:3]: Maybe we ought to use float, I hadn't considered fractional seconds.

**Pull request merged**